src/elections/services/races.service.ts

```diff
@@ -38,29 +37,37 @@ export class RacesService {
     return this.ballotReadyService.fetchRaceNormalizedPosition(raceId)
   }
 
-  async racesByYear({
+  async getRaces({
```

Comment:
Can we rename this `getRacesByZip` to reflect the required input `zip` argument instead of this sounding like a generic "get any races" method?